content/references/markdown/index.md

```diff
@@ -4,12 +4,176 @@ topic: markdown
 next: /references/markdown/extensions/
 ---
 
+
+
 # Basic Markdown Syntax
 
 Codewars supports [Markdown][wiki-markdown].
 
 More specifically, [GitHub Flavored Markdown][gfm] (strict superset of [CommonMark][common-mark]) and few [Codewars extensions][extensions].
 
+You may find below a summary of the most used ones.
+
+
+
+
+## Emphasis
+
+```markdown
+_This is italic_
+*This is also italic*
+
+__This is bold__
+**This is bold too**
+
+___here, italic and bold___
+***here, italic and bold***
+```
+Note that `*You **can** combine them*`: *You **can** combine them*.
+
+
+
+
+## Headers
+
+<pre><code>
+# This is &lt;h1&gt;
+## This is &lt;h2&gt;
+### This is &lt;h3&gt;
+</code></pre>
+
+
+
+## Displaying code
+
+Use code spans and code blocks to show the text as is. Use code block to show the syntax highlighted code.
+
+### Code span
+
+Use single backtick (<code>`</code>) to show the text as is inline.
+
+```markdown
+Code span example: `print("example")`
+```
+
+### Code block
+
+For multiline code blocks, use triple backticks.
+
+Constraints:
+   - Put an empty line before and after the code block itself, so that it's rendered correctly
+   - The triple backticks must be alone one their line (see [extensions](./references/markdown/extensions/) for exceptions)
+
+````markdown
+```
+This is a code block
+       you can indent inside it as you like.
+```
+````
+
+Another way to declare a code block is to put 4 whitesapces at the beginning of the line
+
+````markdown
+   This will be rendered as a code block too
+           you can indent inside it as you like.
+
+````
```

Comment:
```suggestion
```

This is true, but I don't think we need it on this page. Fenced code blocks will be enough.

Reply:
ok. I'll commit after you asnwer to the previous one